src/components/publishing/sections/sections.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react"
+import styled, { StyledFunction } from "styled-components"
+import Header from "../header/header"
+import Authors from "./authors"
+import Embed from "./embed"
+import ImageCollection from "./image_collection"
+import ImagesetPreview from "./imageset_preview"
+import SectionContainer from "./section_container"
+import Text from "./text"
+import Video from "./video"
+
+interface SectionsProps {
+  article: {
+    layout: string
+    authors?: any
+  }
+}
+
+interface StyledSectionsProps {
+  layout: string
+}
+
+const Sections: React.SFC<SectionsProps> = props => {
+  const header = props.article.layout === "standard" ? <Header article={props.article} /> : false
+  return (
+    <StyledSections layout={props.article.layout}>
+      {header}
+      {renderSections(props.article)}
+      {renderAuthors(props.article.authors)}
+    </StyledSections>
+  )
+}
+
+function renderSections(article) {
+  const sections = article.sections
+  const renderedSections = sections.map((section, i) => {
+    switch (section.type) {
+      case "image_collection":
+        return (
+          <SectionContainer key={i} layout={section.layout}>
+            <ImageCollection images={section.images} targetHeight={500} gutter={10} />
+          </SectionContainer>
+        )
+      case "image_set":
+        return (
+          <SectionContainer key={i}>
```

Comment:
lots of repetition of the SectionContainer here, I wonder if the case statement could move inside the container?